### PR TITLE
Include ledger_index in validations subscription stream

### PR DESCRIPTION
**Pull request by wilsonianb**

### CODE REVIEW

**Comment by JoelKatz** [comment]
You have to check if it's present before you include it. It's (technically) optional.

**Comment by wilsonianb** [comment]
Interesting. I didn't know that. I'll add a check.

**Comment by miguelportilla** [comment]
:+1:

File: src/ripple/app/misc/NetworkOPs.cpp [hunk 1]
```diff
@@ -1547,6 +1547,11 @@ void NetworkOPsImp::pubValidation (STValidation::ref val)
         jvObj [jss::ledger_hash]           = to_string (val->getLedgerHash ());
         jvObj [jss::signature]             = strHex (val->getSignature ());
 
+        std::uint32_t seq = val->getFieldU32 (sfLedgerSequence);
```
Comment (vinniefalco) [inline src/ripple/app/misc/NetworkOPs.cpp]: Could be written as
`auto const seq = (*val)[sfLedgerSequence];`

**Comment by vinniefalco** [comment]
Trivial change
:+1:

**Comment by wilsonianb** [comment]
@vinniefalco I tried using `auto` but it resulted in `call to 'to_string' is ambiguous` for
```
jvObj [jss::ledger_index]      = to_string (seq);
```

**Comment by vinniefalco** [comment]
Try
`auto const seq = *(*val)[~sfLedgerSequence];`

**Comment by vinniefalco** [comment]
This decreases test coverage

**Comment by nbougalis** [comment]
Merged as 05c248f297ed2debccd8f7e583b67944c8826820

**Pull request closed without merging**